packages/flutter/lib/src/cupertino/activity_indicator.dart

```diff
@@ -29,11 +38,12 @@ class CupertinoActivityIndicator extends StatefulWidget {
     Key key,
     this.animating = true,
     this.radius = _kDefaultIndicatorRadius,
-  }) : assert(animating != null),
-       assert(radius != null),
-       assert(radius > 0.0),
-       progress = 1.0,
-       super(key: key);
+    this.style = CupertinoActivityIndicatorStyle.iOS13,
```

Comment:
`style` is ambiguous with other unrelated things in Flutter (like [TextStyle](https://master-api.flutter.dev/flutter/cupertino/CupertinoTextField/style.html)).  I think we should rename `style` and `CupertinoActivityIndicatorStyle`.  Maybe to something like `iOSVersionStyle` and `CupertinoActivityIndicatorIOSVersionStyle`?

Reply:
I've renamed it to `iOSVersionStyle`.